docs/spec_draft.md

```diff
@@ -127,3 +127,43 @@ specification.
 %z = stablehlo.add %x, %y : tensor<2x2xf32>
 // %z: [[6, 8], [10, 12]]
 ```
+
+## stablehlo.constant
+
+`stablehlo.constant(value) -> result`
+
+### Semantics
+
+Creates a constant value or array from `value` and produces a `result` tensor.
```

Comment:
What does "value or array" mean here? Perhaps consider rewording to "Creates a `result` tensor from a constant `value`" to avoid the ambiguity?

Reply:
@sdasgup3 made a similar comment earlier, and I'll update the wording to "Produces a `result` tensor from a constant `value`".